src/App.tsx

```diff
@@ -50,15 +40,15 @@
           <div className="tile is-parent">
             <div className="tile is-child box is-success">
               <div className="block">
-                <UserSelector value={author} onChange={setAuthor} />
+                <UserSelector />
               </div>
 
               <div className="block" data-cy="MainContent">
                 {!author && <p data-cy="NoSelectedUser">No user selected</p>}
 
-                {author && !loaded && <Loader />}
+                {author && loaded && <Loader />}
```

Comment:
The condition for rendering the `Loader` component seems incorrect. It should be `author && !loaded` instead of `author && loaded` to display the loader when posts are still loading.